Dockerfile

```diff
@@ -1,27 +1,23 @@
-FROM quay.io/centos/centos:7
+FROM rockylinux:9
```

Comment:
Please use https://quay.io/repository/rockylinux/rockylinux otherwise is easy to incur into docker hub limits.

Reply:
whops, totally forgot that